ui/src/css/_common.scss

```diff
@@ -0,0 +1,80 @@
+.mdc-button {
+  text-transform: none;
+}
+
+.mdc-card__primary-action {
+  padding: 0 8px;
+}
+
+.mdc-card__actions {
+  padding: 0;
+}
+
+.mdc-tab-indicator .mdc-tab-indicator__content--underline {
+  background-color: #1a73e7;
+}
+
+.mdc-typography--headline5 {
+  margin: 0.5em 0;
+}
+
+.mdc-text-field .mdc-text-field__input {
+  caret-color: #757575;
+}
+
+body {
+  margin: 0;
+}
+
+input[type=file] {
+  display: none;
```

Comment:
Yes, it does hide them. I want to include a file upload field so that we can collect photos, but there doesn't seem to be a way to control much of anything about the appearance of a file upload button (not even the button text). So I'm hiding the actual input element, displaying my own regular button, and basically using it as a proxy. The code that's going to do it is here:
https://github.com/nworden/personfinder/blob/nworden-react/ui/src/pages/Create.js#L234
I'll be the first to say it's an ugly solution. I just couldn't find a better one.

Reply:
I see. It may be actually more readable to do that with an inline style instead of a CSS rule i.e.,
```html
<input type="file" style="display: none;"/>
```

Or maybe restrict it to an input with a specific class:
```css
input[type=file].hidden {
  display: none;
}
```
To prevent someone from trying to add a naive `<input type="file">` and be puzzled why it's not visible.

Reply:
Yeah, that's true. I added a class in case we need to do this with other file upload buttons later.